tools/build_versions.py

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env python
 
-import numpy as np
-import scipy as sp
-import matplotlib as mpl
-from PIL import Image
-import Cython
-import networkx
+from pathlib import Path
+import requirements
+import importlib
+import pkg_resources
 
+script_dir = Path(__file__).parent / '..' / 'requirements'
 
-for m in (np, sp, mpl, Image, networkx, Cython):
-    if m is Image:
-        version = m.VERSION
-    else:
-        version = m.__version__
-    print(m.__name__.rjust(10), ' ', version)
+for requirement_file in script_dir.glob('*.txt'):
+    if requirement_file.name == 'numpy.txt':
```

Comment:
Lines 11-13 look redundant to me.

Reply:
good catch. i originally had numpy.txt as a common file between build.txt and defaults.txt, but for some reason it made an other script crash. I ended up reverting the change.